portfolio/src/main/webapp/style.css

```diff
@@ -119,72 +189,156 @@ section.blue {
 }
 
 .card .description {
-  padding: 5px 5px; 
-  text-align: left; 
+  padding: 5px 5px 25px; 
+  text-align: left;
+  position: relative; 
 }
 
-.card h3 {
-  font-size: 25px;
-  font-weight: 500;
-  color: rgb(61, 60, 60);
-  margin-bottom: 2px;
-  transition: .2s;
+.card h4 {
+  font-weight: 600;
+  margin: 15px 0 2px;
+  transition: .4s;
+  color: #0c0c0c;
+  
 }
 
-.card h3 a:hover{
-  color: #84bee6;
+.card h4 a{
+  text-decoration: none;
+  color: inherit;
 }
 
-.card p {
-  font-size: 16px;
-  line-height: 1.6;
-  color: #0C0C0C;
+.card h4 a:hover{
+  color: #a7d7fa;
+}
+
+/*Buttons*/
+.card .button-container {
+  bottom: 20px;
+  position: absolute;
 }
 
 .card button {
   display: block;
   padding: 3px 6px;
-  font-size: 16;
+  margin: 5px;
+  font-size: 15px;
   text-align: center;
   color:#0C0C0C;
   background-color: #c7e8ff;
   border-radius: 10px;
   border: 2px ;
-  box-shadow: 0 2px 5px rgba(0,0,0,.3);
-  transition: .2s;  
+  box-shadow: 0 2px 3px rgba(0,0,0,.3);
+  transition: .1s;  
   float: left;
 }
 
 .card button:hover {
   background-color: #d5eeff;
-  box-shadow: 0 2px 5px rgba(0,0,0,.5);
+  box-shadow: 0 2px 3px rgba(0,0,0,.4);
+}
+
+button {
+  display: block;
+  padding: 6px 8px;
+  margin: 35px auto 5px;
+  font-size: 16px;
+  font: inherit;
+  text-align: center;
+  background-color: #c7e8ff;
+  color: #0C0C0C;
+  border-radius: 13px;
+  transition: .1s;
+  border: none;
 }
 
+button:hover { 
+  background-color: #0c0c0c;
+  color: inherit;
+  border: 2px solid #c7e8ff;
+}
 
+
+#to-do button {
+  display: block;
+  padding: 10px 13px;
+  margin: 15px auto;
+  font: 20px inherit;
+  text-align: center;
+  background-color: #c7e8ff;
+  color: #0C0C0C;
+  border-radius: 15px;
+  transition: .1s;
+  border: none;  
+}
+  
+#to-do button:hover {
+  background-color: #0c0c0c;
+  color: inherit;
+  border: 2px solid #c7e8ff;
+}
+
+#recommendations button {
+  margin: auto auto;
+  text-align: center;
+  border-radius: 8px;
+  width: 100%;
+}
+
+  
 /*Image Class*/
 .pro-pic {
-  width: 150px;
+  height: 175px;
   border:2px solid aliceblue;
   border-radius:50%;
   margin: 10px;
 }
 
+.gallery img {
+  margin-top: 20px;
+  vertical-align: middle;
+  width: 100%;
+  box-shadow: 0 2px 5px rgba(0,0,0,.2);
+  transition: .4s;
+}
+
+.gallery img:hover {
+  transform: translateY(-.8%);
+  box-shadow: 0 4px 10px rgba(0,0,0,.6);
+}
+
 /*Columns*/
+.row {
+  display: flex;
+  display: -ms-flexbox;
+  flex-wrap: wrap;
+  -ms-flex-wrap: wrap;
+}
+
 .column {
-  float: left;
   padding: 20px;
 }
 
+.gallery {
+  padding: 10px;
+}
+
 .column.quarter {
+  flex: 25%;
+  -ms-flex: 25%;
   width: 25%;
```

Comment:
jw why did you set width and flex-basis for these?

Reply:
oh i originally just had width, and then after I added flex i didn't delete. Will delete now!

Reply:
Actually, should I keep both? i just sall in this [http://shouldiprefix.com/#flexbox](url) link you sent, width was kept? Would that apply here?

Reply:
Depends on your goal! If you wanted to make sure your website was browser compatible across the board then yes. But if this is the only place that you are planning on ensuring browser compatibility then it wouldn't be worth it.

Reply:
ah okay, got it!